profile-page.css

```
@@ -0,0 +1,56 @@
+
+.picture{
```

It's better to keep a nice and consistent code style and format - try to train yourself on this attention to detail

```suggestion
.picture {
```